README.md

```diff
@@ -203,6 +203,11 @@ false
 comparing the [Jaccard
 similarity](http://en.wikipedia.org/wiki/Jaccard_index) of two sets.
 
+This implementation includes the improvements recommended in
+"[Improved Densification of One Permutation Hashing]
+(http://arxiv.org/abs/1406.4784)", which greatly reduces the
+time order for building a MinHash.
```

Comment:
I had to read the paper to make sure I knew what you meant by "time order".  The phrase I'm more familiar with is "algorithmic complexity" or "computational complexity" (or, if you can be more specific, "average-case" or "worst-case" complexity).